stdlib.sh

```diff
@@ -133,13 +133,13 @@ direnv_layout_dir() {
 #    log_status "Loading ..."
 #
 log_status() {
-  if [[ -n $DIRENV_LOG_FORMAT ]]; then
-    local msg=$* color_normal=''
-    if [[ -t 2 ]]; then
-      color_normal="\e[m"
-    fi
-    # shellcheck disable=SC2059,SC1117
-    printf "${color_normal}${DIRENV_LOG_FORMAT}\n" "$msg" >&2
+  local msg=$* color_normal=''
+  if [[ -t 2 ]]; then
+    color_normal="\e[m"
+  fi
+  local -r direnv_msg=$(direnv log "$msg" 2>&1)
+  if [[ -n "$direnv_msg" ]]; then
+    printf "${color_normal}%s\n" "${direnv_msg}" >&2
   fi
```

Comment:
Could you move the rest of the logic so that this function can be reduced to:
```
log_status() {
  direnv log -status -- "$*"
}
```
And port the log_error() too, and then finally, both sides will be fully in sync.

Reply:
Sure, sounds good

Reply:
I've made the changes; the only issue was the shade of red used, as it was different in `stdlib.sh` (`"\e[38;5;1m"`) as compared to the colour in go (`\033[31m`).

I used the colour in go.